python/ql/test/experimental/dataflow/regression/test.py

```diff
@@ -179,9 +179,9 @@ def test_truth():
     if t:
         SINK(t)
     else:
-        SINK(t)
+        SINK_F(t)  # False positive
     if not t:
-        SINK(t)
+        SINK_F(t)  # False positive
```

Comment:
My assumption was that `SOURCE = "source"`, but perhaps that is not true in this file...then the test just looks a little odd..

Reply:
@yoff, I forgot about this one...

I think it's very hard to figure out what these old tests are supposed to do, so it could really be either :grimacing: :disappointed:

Reply:
I guess we have to resort to comparing with previous results, then..

Reply:
nicely done :+1:

For the record, yoff found these links:

https://github.com/RasmusWL/codeql/blob/main/python/ql/test/library-tests/taint/general/test.py#L177-L186
https://github.com/RasmusWL/codeql/blob/main/python/ql/test/library-tests/taint/general/TestStep.expected#L197-L200